backend/auth0/management.go

```diff
@@ -0,0 +1,87 @@
+package auth0
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/ansel1/merry/v2"
+	"io"
+	"net/http"
+	"time"
+)
+
+var httpClient = &http.Client{
+	Timeout: time.Second * 5,
+}
+
+func sendManagementRequest[T any](ctx context.Context, client *Client, method, path string, body any) (result T, err error) {
+	var bodyReader io.Reader
+	if body != nil {
+		var stringBody []byte
+		stringBody, err = json.Marshal(body)
+		if err != nil {
+			return
+		}
+		bodyReader = bytes.NewReader(stringBody)
+	}
+
+	managementUrl := fmt.Sprintf("https://%s/api/v2/", client.config.Domain)
+	req, err := http.NewRequestWithContext(ctx, method, fmt.Sprintf("%s%s", managementUrl, path), bodyReader)
+	if err != nil {
+		return
+	}
+
+	token, err := client.GetToken(ctx, client.config.ManagementAudience)
```

Comment:
In general I'm missing "scope" from the code that requests/caches tokens. I assume that means we are getting a token with "all" scopes which isn't really ideal.

Reply:
If necessary. We're using all the scopes we have access to.

Reply:
ok, if the client is only used within this app